.ameba.yml

```diff
@@ -11,18 +11,6 @@ Metrics/CyclomaticComplexity:
 Naming/QueryBoolMethods:
   Enabled: false
 
-Naming/BlockParameterName:
```

Comment:
I'm fine with this rule still being disabled, I've always found it arbitrary and disallows common patterns I use

Reply:
I can disable it again, sure!